src/components/event/EventVisitors/Event-visitors.js

```diff
@@ -20,7 +18,7 @@ class EventVisitors extends Component {
           )}
         />
         <ParticipantGroup
-          disabled={visitors.approvedUsers.length == 0}
+          disabled={visitors.approvedUsers.length === 0}
```

Comment:
Simplify this condition: `disabled={!visitors.approvedUsers.length}`
This comment applies for other similar statements